monkey/tests/unit_tests/infection_monkey/ransomware/test_ransomware_payload.py

```diff
@@ -141,6 +147,26 @@ def test_skip_already_encrypted_file(ransomware_target, ransomware_payload):
     )
 
 
+def test_encryption_skipped_if_configured_false(ransomware_target):
```

Comment:
Shouldn't we also have test for running the encryption using the checkbox ?

Reply:
All of the other tests use the `ransomware_payload_config` fixture which has `should_encrypt` as `True`, i.e. checkbox is checked.

Reply:
We should have a test that verifies no encryption occurs if it's not enabled. See the `test_readme_*()` tests on the develop branch for examples.